packages/common/src/apiutil/LionwebResponse.ts

```diff
@@ -1,5 +1,6 @@
 import { Response } from "express"
 import { LionWebJsonChunk } from "@lionweb/validation";
+import { getRepoVersion } from "./RepoVersion.js";
```

Comment:
Inconsistent ;s, also on line 2. Has `prettier` been run?

Reply:
Now it has